### PR TITLE
fix(hybrid): fix eighth widget format

### PR DESCRIPTION
**Pull request by Laur04**

### CODE REVIEW

**Review by etnguyen03 — APPROVED** [review]
LGTM (currently) but wait until tomorrow to merge (March 17)

**Comment by Laur04** [comment]
Can I merge now?

**Comment by etnguyen03** [comment]
Yes - LGTM

**Pull request merged**
